docs/src/main/sphinx/release/release-470.md

```diff
@@ -0,0 +1,125 @@
+# Release 470 (5 Feb 2025)
+
+## General
+
+* Add [](/connector/duckdb). ({issue}`18031`)
+* Add Loki connector. ({issue}`23053`)
+* Add support for access control systems to control privileges for
+  `CREATE FUNCTION` and `DROP FUNCTION`. ({issue}`24696`)
+* Allow non-lower case variable names in SQL user-defined functions. ({issue}`24460`)
+* Prevent failures of the `ARRAY_HISTOGRAM` function resulting from null values.
+  ({issue}`24765`)
+* Increase compatibility with S3-compliant storage solutions as fault-tolerant
```

Comment:
Improve compatibility of fault tolerant exchange storage with S3-compliant object stores.